nominees/cartodb.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "CartoDB",
+  "description": "CARTO is an open, powerful, and intuitive platform for discovering and predicting the key insights underlying the location data in our world. It empowers organizations to optimize operational performance, strategic investments, and everyday decisions with CARTO Engine— the embeddable platform for web and mobile apps—and the new CARTO Builder, a drag and drop analysis tool.",
+  "website": "https://carto.com/",
+  "license": [
+    {
+      "spdx": "BSD-3-Clause",
+      "licenseURL": "https://github.com/CartoDB/cartodb/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
```

Comment:
@rachita19082 the submission looks good to me apart from it missing at least one of the either `evidenceText` or `evidenceURL`. The preferred choice for `evidenceURL` would be articles, blogs, videoURLs etc. Thanks.